packages/hasher/src/TargetHasher.ts

```diff
@@ -230,7 +231,7 @@ export class TargetHasher {
 
   writeTargetHashesManifest() {
     for (const [id, { fileHashes, globalFileHashes }] of Object.entries(this.targetHashesLog)) {
-      const targetHashesManifestPath = path.join(this.targetHashesDirectory, `${id}.json`);
+      const targetHashesManifestPath = sanitize(path.join(this.targetHashesDirectory, `${id}.json`));
```

Comment:
I think it's better to sanitize just the ID.
```suggestion
      const targetHashesManifestPath = path.join(this.targetHashesDirectory, `${sanitize(id)}.json`);
```